From 13a87c82d38999f58a5e27015b1dbc7cf68bb2c0 Mon Sep 17 00:00:00 2001
From: John Kacur <jkacur@redhat.com>
Date: Fri, 23 Aug 2013 01:31:30 +0200
Subject: [PATCH 3/7] documentation: Install man pages with mode 644

Man pages should not be executable. Install them with mode 644

Signed-off-by: John Kacur <jkacur@redhat.com>
---
 Documentation/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/Makefile b/Documentation/Makefile
index a2ebdc7930e1..c9a6f0960dd6 100644
--- a/Documentation/Makefile
+++ b/Documentation/Makefile
@@ -76,7 +76,7 @@ define do_install
 	if [ ! -d '$(DESTDIR_SQ)$2' ]; then		\
 		$(INSTALL) -d -m 755 '$(DESTDIR_SQ)$2';	\
 	fi;						\
-	$(INSTALL) $1 '$(DESTDIR_SQ)$2'
+	$(INSTALL) -m 644 $1 '$(DESTDIR_SQ)$2'
 endef
 
 MAN1_INSTALL = $(MAN1:%.1=%.1.install)
-- 
1.8.3.1

